### PR TITLE
script for COVID-19 Dataset

### PR DESCRIPTION
**Pull request by ashishpriyadarshiCIC**
script for the data https://www.kaggle.com/kimjihoo/coronavirusdataset/metadata

### CODE REVIEW

**Comment by henrykironde** [comment]
@ashishpriyadarshiCIC this looks good but the links are from `GitHub`. We can either use the kaggle links or we use github and remove the keyword `kaggle true`.  Since this is from one country, we can specify the country name for this dataset instead of `coronavirusdataset`

**Comment by ashishpriyadarshiCIC** [comment]
@henrykironde Fixed it and squashed the commits

File: version.txt [hunk 1]
```diff
@@ -21,6 +21,7 @@ car_eval.json,1.1.2
 catalogos_dados_brasil.json,1.0.0
 chytr_disease_distr.json,1.0.2
 community_abundance_misc.json,1.0.3
+coronaviruskoreadataset.json,1.0.0
```
Comment (henrykironde) [inline version.txt]: Let us change the name to `coronaviruskorea`
south_korea_coronavirus

File: scripts/coronaviruskoreadataset.json [hunk 2]
```diff
@@ -0,0 +1,342 @@
+{
+    "archived": "zip",
+    "citation": "",
+    "description": "Dataset of COVID-19 in South Korea",
+    "homepage": "https://github.com/jihoo-kim/Coronavirus-Dataset",
+    "keywords": [
+        "Github",
+        "Coronavirus",
+        "Covid-19",
+        "Machine Learning"
```
Comment (henrykironde) [inline scripts/coronaviruskoreadataset.json]: Add `south korea` to keywords

File: scripts/coronaviruskoreadataset.json [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+    ],
+    "licenses": [
+        {
+            "name": "CC BY-NC-SA 4.0"
+        }
+    ],
+    "name": "coronaviruskoreadataset",
```
Comment (henrykironde) [inline scripts/coronaviruskoreadataset.json]: south-korea-coronavirus

File: scripts/coronaviruskoreadataset.json [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+                    {
+                        "name": "latitude",
+                        "size": "9",
+                        "type": "char"
+                    },
+                    {
+                        "name": "longitude",
+                        "size": "10",
+                        "type": "char"
+                    }
```
Comment (henrykironde) [inline scripts/coronaviruskoreadataset.json]: `latitude, longitude` should not be char

Comment (ashishpriyadarshiCIC) [reply scripts/coronaviruskoreadataset.json]: Fixed it :+1:

File: scripts/coronaviruskoreadataset.json (hunk 3 above)
Comment (ashishpriyadarshiCIC) [reply scripts/coronaviruskoreadataset.json]: Changed it

File: scripts/coronaviruskoreadataset.json (hunk 2 above)
Comment (ashishpriyadarshiCIC) [reply scripts/coronaviruskoreadataset.json]: Done :+1:

File: version.txt (hunk 1 above)
Comment (ashishpriyadarshiCIC) [reply version.txt]: Fixed it, I agree coronaviruskoreadataset was a complicated name for the dataset

**Comment by henrykironde** [comment]
We are going to wait for the kaggle support to be merged and then we shall test this PR

**Comment by ashishpriyadarshiCIC** [comment]
@henrykironde I might also have to make changes as the data was updated 3 hours ago in which some more files were added.

File: scripts/coronaviruskorea.json [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+        "Coronavirus",
+        "Covid-19",
+        "Machine Learning"
+    ],
+    "licenses": [
+        {
+            "name": "CC BY-NC-SA 4.0"
+        }
+    ],
+    "name": "south_korea_coronavirus",
```
Comment (henrykironde) [inline scripts/coronaviruskorea.json]: south-korea-coronavirus

File: version.txt [hunk 6]
```diff
@@ -21,6 +21,7 @@ car_eval.json,1.1.2
 catalogos_dados_brasil.json,1.0.0
 chytr_disease_distr.json,1.0.2
 community_abundance_misc.json,1.0.3
+coronaviruskorea.json,1.0.0
```
Comment (henrykironde) [inline version.txt]: this file name should be `south_korea_coronavirus.json`

**Comment by henrykironde** [comment]
@ashishpriyadarshiCIC let's rename to coronavirus-south-korea.
We should make sure that all the corona data has the same naming style 
https://github.com/weecology/retriever-recipes/pull/60 @DumbMachine this applies to the Italian dataset too.

**Comment by ashishpriyadarshiCIC** [comment]
@henrykironde Made all the changes and updated the links for the data

**Comment by ashishpriyadarshiCIC** [comment]
This source of this data is https://github.com/ThisIsIsaac/Data-Science-for-COVID-19 but it is being frequently updated in Kaggle instead of GitHub so the links from Kaggle are being used for this dataset.

**Comment by henrykironde** [comment]
Thanks for the work @ashishpriyadarshiCIC we have merged this after some changes 
ref https://github.com/weecology/retriever-recipes/pull/89

**Pull request closed without merging**